packages/resource-table/src/ResourceTable.tsx

```diff
@@ -77,67 +80,76 @@ const ResourceTable = <ResourceType,>({
     return columns;
   }, [selectedResource, detailsTabs, primaryColumnKeys]);
 
+  const filteredResources = useMemo(
+    () => (filter ? resources.filter(filter) : resources),
+    [resources, filter],
+  );
+
   useEffect(() => {
     if (selectedResource) {
       onRowSelection?.(selectedResource);
     }
   }, [selectedResource]);
 
   return (
-    <ThemeProvider theme={'chrome'} colorScheme={colorScheme}>
-      <div className="rq-resource-table-container" data-scheme={colorScheme}>
-        <SplitPane className="rq-resource-table-splitpane">
-          <div onScroll={onScroll} ref={scrollableContainerRef}>
-            <Table
-              className="rq-resource-table"
-              selected={selectedRowId}
-              onSelected={setSelectedRowId}
-            >
-              <Table.Head>
-                <Table.Row>
-                  {columnsToRender.map((column) => (
-                    <Table.HeadCell
-                      key={column.key}
-                      style={{ width: column.width ? `${column.width}%` : 'auto' }}
-                    >
-                      {column.header}
-                    </Table.HeadCell>
-                  ))}
-                </Table.Row>
-              </Table.Head>
-              <ContextMenu
-                items={contextMenuOptions}
-                selectedResource={contextMenuSelectedResource}
-                handleOpenChange={onContextMenuOpenChange}
-              >
-                <Table.Body>
-                  {resources.map((resource, index) =>
-                    !filter || filter(resource) ? (
-                      <ResourceTableRow
-                        key={index}
-                        id={getRowId(index)}
-                        resource={resource}
-                        columns={columnsToRender}
-                        isFailed={isFailed}
-                        setContextMenuSelectedResource={setContextMenuSelectedResource}
-                      />
-                    ) : null,
-                  )}
-                </Table.Body>
-              </ContextMenu>
-            </Table>
+    <>
+      {!filteredResources.length && emptyView ? (
+        { emptyView }
```

Comment:
This breaks, since we are returning an object, emptyView is an react node, so we can directly return it.
```diff
- { emptyView }
+ emptyView 
```

Reply:
Also can we give some default style to it ie `emptyView` should always be at the center of the container also give some padding around it.

Example:
<img width="1109" alt="Screenshot 2023-09-27 at 5 48 53 PM" src="https://github.com/requestly/requestly-ui/assets/61556757/ab6a2bbf-f818-4c08-b510-c5908c65a8df">